src/Samples/DIPS.Xamarin.UI.Samples/Controls/FloatingActionMenu/FloatingActionMenuPage.xaml

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0"
+      encoding="utf-8"?>
+
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:dxui="http://dips.xamarin.ui.com"
+             xmlns:FloatingActionMenu="clr-namespace:DIPS.Xamarin.UI.Samples.Controls.FloatingActionMenu;assembly=DIPS.Xamarin.UI.Samples"
+             mc:Ignorable="d"
+             x:Class="DIPS.Xamarin.UI.Samples.Controls.FloatingActionMenu.FloatingActionMenuPage">
+    <ContentPage.Content>
+        <RelativeLayout>
+            <RelativeLayout.Behaviors>
+                <dxui:FloatingActionMenuBehaviour>
```

Comment:
Didn't we agree on reusing ModalityLayout?

Reply:
After we talked about the intended use of the ModalityLayout I decided to do it like this instead. It also removes the need to use a modality layout when an absolute layout might make more sense. Now it works for both